content/drone-plugins/drone-docker/index.md

```diff
@@ -122,6 +122,17 @@ pipeline:
     secrets: [ docker_username, docker_password ]
 ```
 
+Example configuration that allow build a specific target:
```

Comment:
Can you elaborate a bit more on this example? Just saying "target" may not be very clear to new users.

Reply:
@tonglil thx!
Do you think that add a link to https://docs.docker.com/develop/develop-images/multistage-build/#name-your-build-stages should be fine or I should include more elaborate docs about multistage build on docker?

Reply:
Yes, giving some context would be good (link and/or otherwise).